Source/JavaScriptCore/builtins/PromiseOperations.js

```diff
@@ -452,7 +454,15 @@ function promiseReactionJob(promiseOrCapability, handler, argument, contextOrSta
 
     // Case (4).
     if (!promiseOrCapability) {
-        @promiseReactionJobWithoutPromise(handler, argument, contextOrState);
+        // inlined promiseReactionJobWithPromise
+        try {
+            if (contextOrState)
+                handler(argument, contextOrState);
+            else
+                handler(argument);
+        } catch {
+            // This is user-uncatchable promise. We just ignore the error here.
+        }
```

Comment:
Based on the discussion on Slack (https://webkit.slack.com/archives/CTV4FGWF4/p1756915196720469), I'm manually inlining the call to `@promiseReactionJobWithoutPromise` by `@promiseReactionJob`. When I ran JetStream/doxbee-async locally, this indeed improves the score.